File: src/com/yijiwenhua/backend/model/SysCompany.java
package com.yijiwenhua.backend.model;

import java.io.Serializable;

/**
 * sys_company
 * @author lvliang
 * @since  2018-07-09
 */
public class SysCompany implements Serializable{
	private static final long serialVersionUID = 1L;
	
	/**id*/
	private Integer id;
	
	/**公司名称*/
	private String name;
	
	/**公司地址*/
	private String address;
	
	/**公司logo*/
	private String logo;
	
	/**电话*/
	private String mobile;
	
	/**邮箱*/
	private String email;
	
	/**公司描述*/
	private String desc;
	
	/**id*/
	public Integer getId() {
		return id;
	}
	
	public void setId(Integer id) {
		this.id = id;
	}
	
	/**公司名称*/
	public String getName() {
		return name;
	}
	
	public void setName(String name) {
		this.name = name;
	}
	
	/**公司地址*/
	public String getAddress() {
		return address;
	}
	
	public void setAddress(String address) {
		this.address = address;
	}
	
	/**公司logo*/
	public String getLogo() {
		return logo;
	}
	
	public void setLogo(String logo) {
		this.logo = logo;
	}
	
	/**电话*/
	public String getMobile() {
		return mobile;
	}
	
	public void setMobile(String mobile) {
		this.mobile = mobile;
	}
	
	/**邮箱*/
	public String getEmail() {
		return email;
	}
	
	public void setEmail(String email) {
		this.email = email;
	}
	
	/**公司描述*/
	public String getDesc() {
		return desc;
	}
	
	public void setDesc(String desc) {
		this.desc = desc;
	}
	
}

File: WebRoot/static/validatejs/SysCompany.js
var saveSysCompanyConfig = {
	name:[{rule:notBlank,msg:"公司名称 不能为空"},{rule:length,params:{min:0,max:50},msg:"公司名称 长度不在0-50范围"}],
	address:[{rule:notBlank,msg:"公司地址 不能为空"},{rule:length,params:{min:0,max:200},msg:"公司地址 长度不在0-200范围"}],
	logo:[{rule:notBlank,msg:"公司logo 不能为空"}],
	covers:[{rule:notBlank,msg:"网站封面 不能为空"}],
	peripherys:[{rule:notBlank,msg:"公司周边 不能为空"}],
	mobile:[{rule:notBlank,msg:"公司电话 不能为空"},{rule:length,params:{min:0,max:20},msg:"电话 长度不在0-20范围"}],
	email:[{rule:notBlank,msg:"公司邮箱 不能为空"},{rule:length,params:{min:0,max:50},msg:"邮箱 长度不在0-50范围"}],
	desc:[{rule:notBlank,msg:"公司描述 不能为空"},{rule:length,params:{min:0,max:65535},msg:"公司描述 长度不在0-65535范围"}]
};
